models/chainlink/bnb/chainlink_bnb_ocr_fulfilled_transactions.sql

```diff
@@ -0,0 +1,53 @@
+{{
+  config(
+    tags=['dunesql'],
+    alias='ocr_fulfilled_transactions',
+    materialized='view',
```

Comment:
you materialize as a view, yet use incremental logic below. this won't do anything. it'll simply ignore all incremental logic. you should either:
- make this incremental, then add the other config properties needed (file type delta, unique keys, merge approach, etc) -- there are plenty of examples in spellbook to follow
- keep as view, but remove anything incremental in the model